apps/managedidentity/managedidentity.go

```diff
@@ -307,8 +317,9 @@ func (c Client) AcquireToken(ctx context.Context, resource string, options ...Ac
 		option(&o)
 	}
 	c.authParams.Scopes = []string{resource}
+	c.logger.Log(ctx, slog.LevelInfo, "Managed Identity", slog.String("source", string(c.source)))
```

Comment:
This will log the source upon every token request. While it's definitely helpful to log the source, you really only want to do so once because the value is immutable. I guess you placed this here instead of in `New` because `New` doesn't get context? This is pretty awkward 😆. I suppose the possible solutions are
- create a context (with timeout) in `New` for logging the source
- log this here but only once per process (see [sync.Once](https://pkg.go.dev/sync#Once))

Option 2 looks better to me